packages/ingest/granule.js

```diff
@@ -156,13 +156,15 @@ class Granule {
    * @param {Object} provider - provider configuration object
    * @param {string} fileStagingDir - staging directory on bucket to place files
    * @param {boolean} forceDownload - force download of a file
+   * @param {boolean} duplicateHandling - specify how to handle duplicates
    */
   constructor(
     buckets,
     collection,
     provider,
     fileStagingDir = 'file-staging',
-    forceDownload = false
+    forceDownload = false,
+    duplicateHandling = 'replace'
```

Comment:
After a quick glance at `ingest/granule.js`, I don't see any behavior defined for when duplicateHandling is set to "replace".  Am I not looking in the right place?

Reply:
There seems to be a lot of places where duplicateHandling is set to 'replace' in the Cumulus test data.  I do not believe that we added this field.